src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/ContactDiaryStore.swift

```diff
@@ -290,17 +317,30 @@ class ContactDiaryStore: DiaryStoring, DiaryProviding {
 			let sql = """
 				INSERT INTO ContactPersonEncounter (
 					date,
-					contactPersonId
+					contactPersonId,
+					duration,
+					maskSituation,
+					setting,
+					circumstances
 				)
 				VALUES (
 					date(:dateString),
-					:contactPersonId
+					:contactPersonId,
+					:duration,
+					:maskSituation,
+					:setting,
+					:circumstances
 				);
 			"""
 
 			let parameters: [String: Any] = [
 				"dateString": date,
-				"contactPersonId": contactPersonId
+				"contactPersonId": contactPersonId,
+				// CJE: Handle Optionals
+//				"duration": duration?.rawValue,
+//				"maskSituation": maskSituation?.rawValue,
+//				"setting": setting?.rawValue,
```

Comment:
Is this still to be handled?

Reply:
Will be handled in another branch, as mentioned in the description 😁